presto-main/src/main/java/com/facebook/presto/server/QueryStateInfo.java

```diff
@@ -147,4 +152,10 @@ public String getUser()
     {
         return resourceGroupChain;
     }
+
+    @JsonProperty
+    public DateTime getCreateTime()
+    {
+        return createTime;
```

Comment:
Is `DateTime` problematic? We already use in other places (e.g. `com.facebook.presto.execution.QueryStats#getCreateTime`)

(I only hope the json representation is independent from the fact we use joda)